code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -319,7 +297,12 @@
 		cooldown = COOLDOWN_DAMAGE
 		for(var/V in listeners)
 			var/mob/living/L = V
-			L.apply_damage(15 * power_multiplier, def_zone = BODY_ZONE_CHEST, wound_bonus=CANT_WOUND)
+			if(ismegafauna(L) || istype(L, /mob/living/simple_animal/hostile/asteroid))
+				L.apply_damage(360 * power_multiplier, def_zone = BODY_ZONE_CHEST, wound_bonus=CANT_WOUND) // very long cooldown so make it really good at killing lavaland mobs
+			else if(L.mind?.has_martialart(MARTIALART_ULTRAVIOLENCE))
+				L.apply_damage(45 * power_multiplier, def_zone = BODY_ZONE_CHEST, wound_bonus=CANT_WOUND) // DIE!!
+			else
+				L.apply_damage(15 * power_multiplier, def_zone = BODY_ZONE_CHEST, wound_bonus=CANT_WOUND)
```

Comment:
instead of having 3 different apply_damage calls, create a damage variable and modify it in the if elses then use it in a single apply_damage